sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=process.env.SENTRY_ORG
+defaults.project=process.env.SENTRY_PROJECT
+cli.executable=../../.npm/_npx/a8388072043b4cbc/node_modules/@sentry/cli/bin/sentry-cli
```

Comment:
I'm not convinced this looks correct. I think this will work on your local env but not elsewhere. I will try this locally to confirm

Reply:
Yes, we're not utilizing this file currently, thus I'll remove it from the codebase, thank you!